### ISSUE TITLE
Add support for kitty terminal:  WARNING: terminal is not fully functional

### ISSUE THREAD

**Issue by yonas** [dalance/procs#316]
`WARNING: terminal is not fully functional`

Setting TERM=xterm-color makes this go away.

**Comment by MithicSpirit** [comment]
What is your `TERM` set to when the warning shows up?

**Comment by yonas** [comment]
@MithicSpirit It was set to `xterm-kitty`

**Comment by MithicSpirit** [comment]
I believe that the issue is that `xterm-kitty` is not your terminfo database. The correct `TERM` for kitty should just be `kitty`; please try running `TERM=kitty procs` to see if it works.

Also, this message is probably being emitted by `less` (the pager) itself, rather than `procs`. You can try running something like `echo foobar | less` to check whether that is the case.

**Comment by yonas** [comment]
@MithicSpirit 

> I believe that the issue is that `xterm-kitty` is not your terminfo database. The correct `TERM` for kitty should just be `kitty`; please try running `TERM=kitty procs` to see if it works.

Running `TERM=kitty procs` doesn't fix the issue.

I have `/usr/share/terminfo/x/xterm-kitty` installed with sha256 as `dcc9cd047af252561521ed8e9e4879569987f626a316ddf008239ef3bf2d65d3`.

Ubuntu 22.04.1 LTS
uname: `Linux desktop 5.15.0-47-generic #51-Ubuntu SMP Thu Aug 11 07:51:15 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`
 
> Also, this message is probably being emitted by `less` (the pager) itself, rather than `procs`. You can try running something like `echo foobar | less` to check whether that is the case.

Running `echo foobar | less` doesn't produce any warnings.

**Comment by MithicSpirit** [comment]
Oh, you're on Ubuntu, are you perhaps using the `procs` as a snap (or any other form of containerization)?

**Comment by yonas** [comment]
@MithicSpirit Yes, I'm using `procs` v0.13.0 as a snap.

**Comment by MithicSpirit** [comment]
I think that the issue is that `xterm-kitty` does not exist in the terminfo database in the snap image.

**Comment by yonas** [comment]
@MithicSpirit You're right. It works when building it via `cargo install procs`. Somehow the snap needs to be updated with term info or get terminfo from outside it's container (which could be counter-productive).

I don't know how snaps are created, so I'll have to leave this with you. Thanks!

**Comment by thisismygitrepo** [comment]
I faced similar problems with many other tools like ssh, image viewer etc.
It became evident to me that it was kitty to blame not the tools.
It was a long time since I dumped it and migrated to `wezterm`. None of this happen.
